paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -133,12 +133,14 @@ export interface WriteIdsAndPrefsOptions extends Options {
   redirectUrl?: URL;
 }
 
+export interface CreateSeedOptions extends Options {
```

Comment:
Would it be more aligned with PrebidJS code if  add ab error management with an extra callback? For instance, in PrebidJS, I see functions including more than one callback. https://github.com/openx/Prebid.js/blob/master/src/adapters/bidderFactory.js#L281

E.g:
```
onSuccess?: (s: Seed) => void;
onError?: (reason: string) => void;
```